test/cypress/integration/users-roles.spec.js

```diff
@@ -0,0 +1,43 @@
+import { generateRandomId } from '../support/utils';
+
+describe('Roles', () => {
+  const roleCode = generateRandomId();
+
+  beforeEach(() => {
+    cy.appBuilderLogin();
+  });
+
+  afterEach(() => {
+    cy.appBuilderLogout();
+  });
+/*
```

Comment:
please remove commented code (but I'm not sure what users-roles will contain then)